docs/adminguide/updating/index.rst

```diff
@@ -0,0 +1,49 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+.. _doc_adminguide_updating:
+
+.. cspell:words linenos literalinclude
+
+======================
+Parsec Server updating
+======================
+
+This guide covers the updating procedure for the `Parsec`_ server (also called `parsec-backend`).
+
+.. note::
+
+  We consider that you are using the same infrastructure configuration as in the :ref:`Parsec Server installation guide <doc_adminguide_hosting>`.
+  Adapt this guide to your own situation.
+
+.. _Parsec: https://parsec.cloud
+
+Updating to a newer version
+***************************
+
+.. warning::
+
+  Before updating to a newer version, we recommend that you backup the database,
+  In case you need to rollback the update.
+
+In this guide, we will consider that you are using the parsec-backend at the version ``2023-10-11-v2.16.0-rc.5.dev-b33d909`` and you want to update-it to ``v2.16.0``.
```

Comment:
I think the guide should show how to migrate from a public version to another (For example from `2.15.0` to `2.16.0`) since this is the most probable scenario for Parsec users.

Reply:
We didn't have a docker image in `2.15` :/

But I can do it for the form

Reply:
(partially resolved)

Reply:
done